Documentation/configuring-storage.md

```diff
@@ -56,20 +56,64 @@ The `spec.storage.hive.s3Compatible.secretName` should be the name of a secret i
 
 > **Note**: The values of the `aws-access-key-id` and `aws-secret-access-key` must be base64 encoded.
 
-For example:
-
+Noobaa installation example:
+1. get noobaa CLI
+     * download [noobaa CLI tool](https://github.com/noobaa/noobaa-core) and put it into your bin PATH
+2. install noobaa in your cluster
+```bash
+oc create ns noobaa
+oc project noobaa
+noobaa install
+```
+3. After a successful noobaa installation, we need to create a bucket to store the reporting data
+```bash
+noobaa bucket create foobar
+```
+4. extract the aws access information from the noobaa secret
+```bash
+oc get secret noobaa-admin -n noobaa -o json | jq '.data|map_values(@base64)'
+{
+  "AWS_ACCESS_KEY_ID": "<BASE64_ENCODED_KEY_ID>",
+  "AWS_SECRET_ACCESS_KEY": "<BASE64_ENCODED_ACCESS_KEY>",
+  "email": "<BASE64_ENCODEDD_EMAIL>",
+  "password": "<BASE64_ENCODEDD_PASSWORD>",
+  "system": "<BASE64_ENCODED_VALUE>"
+}
+```
+5. create secret based on step 4.
 ```yaml
 apiVersion: v1
 kind: Secret
 metadata:
-  name: your-aws-secret
+  name: noobaa-aws-secret
 data:
-  aws-access-key-id: "dGVzdAo="
-  aws-secret-access-key: "c2VjcmV0Cg=="
+  aws-access-key-id: "<BASE64_ENCODED_KEY_ID_FROM_STEP_4>"
+  aws-secret-access-key: "<BASE64_ENCODED_ACCESS_KEY_FROM_SETP_4>"
 ```
-
 To store data in S3 compatible storage, the `aws-access-key-id` and `aws-secret-access-key` credentials must have read and write access to the bucket.
 
+6. create meteringconfig
```

Comment:
For Steps 6 and 7, s/meteringconfig/MeteringConfig. In the OCP docs, we initial capitalize this resource as MeteringConfig. Disregard if this is not the same for these upstream docs.